src/Microsoft.ML.PCA/PcaTransformer.cs

```diff
@@ -670,8 +629,74 @@ internal static class Defaults
             public const int Seed = 0;
         }
 
+        /// <summary>
+        /// Describes how the transformer handles one column pair.
+        /// </summary>
+        public sealed class ColumnInfo
+        {
+            /// <summary>
+            /// Name of the column resulting from the transformation of <see cref="InputColumnName"/>.
+            /// </summary>
+            public readonly string Name;
+            /// <summary>
+            /// Name of column to transform.
+            /// If set to <see langword="null"/>, the value of the <see cref="Name"/> will be used as source.
+            /// </summary>
+            public readonly string InputColumnName;
+            /// <summary>
+            /// The name of the weight column.
+            /// </summary>
+            public readonly string WeightColumn;
+            /// <summary>
+            /// The number of components in the PCA.
+            /// </summary>
+            public readonly int Rank;
+            /// <summary>
+            /// Oversampling parameter for randomized PCA training.
+            /// </summary>
+            public readonly int Oversampling;
+            /// <summary>
+            /// If enabled, data is centered to be zero mean.
+            /// </summary>
+            public readonly bool Center;
+            /// <summary>
+            /// The seed for random number generation.
+            /// </summary>
+            public readonly int? Seed;
+
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="name">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+            /// <param name="inputColumnName">Name of column to transform.
+            /// If set to <see langword="null"/>, the value of the <paramref name="name"/> will be used as source.</param>
+            /// <param name="weightColumn">The name of the weight column.</param>
+            /// <param name="rank">The number of components in the PCA.</param>
+            /// <param name="overSampling">Oversampling parameter for randomized PCA training.</param>
+            /// <param name="center">If enabled, data is centered to be zero mean.</param>
+            /// <param name="seed">The seed for random number generation.</param>
```

Comment:
>The seed for random number generation. [](start = 35, length = 38)

    /// <param name="seed">The random seed. If unspecified random state will be instead derived from the <see cref="MLContext"/>.</param>